cparser/Parser.vy

```diff
@@ -36,7 +36,7 @@ Require Import List.
   CHAR SHORT INT LONG SIGNED UNSIGNED FLOAT DOUBLE CONST VOLATILE VOID
   STRUCT UNION ENUM UNDERSCORE_BOOL PACKED ALIGNAS ATTRIBUTE ASM
 
-%token<cabsloc> CASE DEFAULT IF ELSE SWITCH WHILE DO FOR GOTO CONTINUE BREAK
+%token<cabsloc> CASE DEFAULT IFt ELSE SWITCH WHILE DO FOR GOTO CONTINUE BREAK
```

Comment:
I guess `IF` now conflicts with something?  Coq's notation? Something else?

Nothing serious but I would have a slight preference for `IF_` over `IFt`.